modules/tinymce/CHANGELOG.md

```diff
@@ -6,7 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
-## Fixed
+### Added
+- Typography added to the default menubar tools list. #TINY-9450
```

Comment:
I don't think this is quite the right wording.

Maybe this previous example is more what we normally write: 

> - Added missing code menu item from the default menu config. #TINY-1648

I think there's also been a previous argument that we don't mention premium-only features in the changelog, as it isn't relevant to the community version. Not sure about that, though.

Reply:
That second point is the important one.

- [ ] Please revert the changelog

Reply:
Reverted!